### PR TITLE
Change "My" to "Manage" in all copy for teacher tabs

### PR DESCRIPTION
**Pull request by cissyxyu**
## WHAT
Rename the teacher tabs "My []" to "Manage []" and "My Reports" to "View Reports".

## WHY
This is more in line with standard naming in ed-tech apps.

## HOW
Just copy changes -- change the string constants to reflect the new naming.

### Screenshots
Screenshot 2024-08-06 at 9 48 28 PM

### Notion Card Links
https://www.notion.so/quill/1-Update-Names-for-Primary-Navigation-Tabs-7484e4a03ba84253b50e85cdd1bd2a03?pvs=4

### What have you done to QA this feature?
Deploy to staging and verify that teachers see the new copy on all front end.

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  yes
Have you deployed to Staging? | yes
Self-Review: Have you done an initial self-review of the code below on Github? | yes

### CODE REVIEW

File: services/QuillLMS/app/helpers/navigation_helper.rb [hunk 1]
```diff
@@ -18,9 +18,9 @@ module NavigationHelper
   LOG_IN = 'Log In'
   LOG_OUT = 'Log Out'
   MY_ACCOUNT = 'My Account'
-  MY_ACTIVITIES = 'My Activities'
-  MY_CLASSES = 'My Classes'
-  MY_REPORTS = 'My Reports'
+  MY_ACTIVITIES = 'Manage Activities'
+  MY_CLASSES = 'Manage Classes'
+  MY_REPORTS = 'View Reports'
```
Comment (eadams17) [inline services/QuillLMS/app/helpers/navigation_helper.rb]: we should change these const names to match the strings

also, in this same file, we should update the definition names (i.e. `my_classes_tab`, etc)

**Review by eadams17 — CHANGES_REQUESTED** [review]
This looks good-- just some clean up I think we should do within the `navigation_helper` file to match the new naming convention

**Review by eadams17 — APPROVED** [review]

**Review by anathomical — APPROVED** [review]
I appreciate the updates to the variable names that you included as part of this.

**Review by emilia-friedberg — APPROVED** [review]
LGTM!

**Pull request merged**